### DISCUSSION TITLE
Can you send variables?

### DISCUSSION THREAD

**Discussion by dslugPX** [toverainc/willow#144] [General]
I kind of asked this in another discussion that didn't take off - but figured I'd this question front and center. 

Is there any facility to do things like say: 

"Hi ESP, turn the volume up $some_value"

I don't quite get the full breadth of capability of the inference server yet. I know how do stuff like this with scripting in Home Assistant using text, but I haven't even began to consider how to manage such a thing with Willow "talking" to Home Assistant for me. 

Is this something that:
A: You can do right now!
B: Is absolutely going to be possible but we didn't get there yet.
C: Might be possible, but is super hard to think about right now and we haven't even started to really.
D: Ain't never gonna happen bub!
or 
E: Something DXPlug hasn't thought of yet.

My money is on E.

thanks!

**Comment by stintel** [comment]
If you're using WIS, the answer is A.
